src/app/core/user/user-select/user-select.component.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewChild,
+} from "@angular/core";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { User } from "../user";
+import { MatAutocompleteTrigger } from "@angular/material/autocomplete";
+import { AlertService } from "../../alerts/alert.service";
+import { LoggingService } from "../../logging/logging.service";
+
+@Component({
+  selector: "app-user-select",
+  templateUrl: "./user-select.component.html",
+  styleUrls: ["./user-select.component.scss"],
+})
+export class UserSelectComponent implements OnInit {
+  @Input() user: string;
```

Comment:
To make this more generally powerful we should probably use `User` entity objects (or maybe id) in the input and output. Then the code using this component can decide which fields are needed at a higher level.

We'll probably need to do a simple migration of old data, mapping names to ids and save user ids with the note, but that should not be a big issue.

Reply:
So to just to clarify, a Note should therefore not only have one author (which currently is a string) but multiple authors (which should be an array of users, resp. an array of user IDs)?

Reply:
Yes, that should be the it.

And maybe we can abstract this (now or later) into a generic base component also, so the UX and implementation is shared between child and user select field (#136)